ui/raidboss/data/05-shb/trial/wol-ex.js

```diff
@@ -0,0 +1,368 @@
+'use strict';
+
+[{
+  zoneId: ZoneId.TheSeatOfSacrificeExtreme,
+  timelineFile: 'wol-ex.txt',
+  triggers: [
+    {
+      id: 'WOLEx Terror Unleashed',
+      netRegex: NetRegexes.ability({ source: 'Warrior Of Light', id: '4F09', capture: false }),
+      condition: function(data) {
+        return data.role === 'healer';
+      },
+      suppressSeconds: 5,
+      alertText: {
+        en: 'Full Heal Everyone',
+      },
+    },
+    {
+      id: 'WOLEx To The Limit',
+      netRegex: NetRegexes.startsUsing({ source: 'Warrior Of Light', id: '4F3[456]' }),
+      run: function(data, matches) {
+        data.limitBreaks = {
+          en: {
+            0: 'role positions',
+            1: 'stacks',
+            2: 'meteor',
+          },
+        }[data.displayLang];
+        if (matches.id == '4F34')
+          data.limitBreak = data.limitBreaks[0];
+        if (matches.id == '4F35')
+          data.limitBreak = data.limitBreaks[1];
+        if (matches.id == '4F36')
+          data.limitBreak = data.limitBreaks[2];
+      },
+    },
+    {
+      // TODO: Replace with Timeline Trigger for earlier alerting
+      id: 'WOLEx Limit Break',
+      netRegex: NetRegexes.startsUsing({ source: 'Warrior Of Light', id: ['4EFB', '515C', '53CB'], capture: false }),
+      alertText: function(data) {
+        let msg = data.limitBreak;
+        delete data.limitBreak;
+        return msg;
+      },
+    },
+    {
+      id: 'WOLEx Absolute Stone III',
+      netRegex: NetRegexes.startsUsing({ source: 'Warrior Of Light', id: '4F2C', capture: false }),
+      infoText: {
+        en: 'Protean',
+      },
+    },
+    {
+      id: 'WOLEx Imbued Absolute Fire III',
+      netRegex: NetRegexes.startsUsing({ source: 'Warrior Of Light', id: '4EF3', capture: false }),
+      run: function(data) {
+        data.imbued = data.imbued || [];
+        data.imbuedFire = {
+          en: 'Stop',
```

Comment:
Here and elsewhere, is it worth dropping the current translations from `responses` in, given that it's already done? Might be nice for the users who are running between now and when the translators get to this. (Obviously it's going to be imperfect given that non-Latin languages might not map correctly with the `+` joining, but it's something at least, right?)

Reply:
My hope is that we can push Responses.X to the stack and then instead of calling `alertText:` call `response:`? I don't know if that's how that works so I wanted to consult with @quisquous first. But yea, failing that, we're copying/pasting and that can be done either in this PR or following up.

Reply:
Right, so far as I know that's not currently supported, I've just been doing copypasta for these situations in my own work. But I would be extremely happy to be wrong!

Reply:
Sounds like no to my hope of chaining responses, but like you wrote it's a common issue that we can address. But, reading the responses more they're also a little long (ie more than a single word) so they're hard to chain together instead of "Stop + Out + Stack" you get like "Stop everything!! + Get Out + Stack" which isn't as succinct as the callout should be.

General translations pushed.